controllers/devworkspace/util.go

```diff
@@ -0,0 +1,26 @@
+package devworkspace
+
+import (
+	chev2alpha1 "github.com/eclipse-che/che-operator/api/v2alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type DevworkspaceState int
+
+const (
+	DevworkspaceStateNotPresent = 0
```

Comment:
```suggestion
	DevworkspaceStateNotPresent DevworkspaceState = 0
	DevworkspaceStateDisabled   DevworkspaceState = 1
	DevworkspaceStateEnabled    DevworkspaceState = 2
```
it would be a bit better from go typing perspective.